subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -1297,44 +1297,21 @@ psa_status_t cracen_export_key(const psa_key_attributes_t *attributes, const uin
 			       size_t *data_length)
 {
 #ifdef CONFIG_PSA_NEED_CRACEN_KMU_DRIVER
-	int status;
 	psa_key_location_t location =
 		PSA_KEY_LIFETIME_GET_LOCATION(psa_get_key_lifetime(attributes));
 
 	if (location == PSA_KEY_LOCATION_CRACEN_KMU) {
-		/* The keys will already be in the key buffer as they got loaded their by a previous
```

Comment:
Nit but I would propose to keep both of the comments regarding how they keys are loaded and the decision about which mutex to use.